components/WorkDatesDisplay/WorkDatesDisplay.styled.js

```diff
@@ -0,0 +1,39 @@
+import styled from "styled-components";
+
+export const WorkDatesContainer = styled.div`
```

Comment:
please try to use some more semantic html tags like article, section, main. etc.